requirements.txt

```diff
@@ -7,5 +7,5 @@ cryptography >= 2.9.2
 pg8000 >= 1.15.3
 pyparsing >= 2.4.7
 python-tds >= 1.10.0
-gunicorn >= 20.0.4
 flask_cors >= 1.1
+waitress >= 1.4.4
```

Comment:
Since `waitress` is now the default I think it'd be fine to remove `gunicorn` from the dependencies and just leave it as a heavy-request env optional that people can install on their own.